src/app/app.component.ts

```diff
@@ -141,6 +141,14 @@ export class AppComponent implements OnInit, OnDestroy {
     )
   }
 
+  showDriverTrips() {
+    return (
+      remult.isAllowed(Roles.trainee) ||
+      getSite().allowAnyVolunteerToAdd // hack for now
+      //TODO showCopyLink needed? What is it?
```

Comment:
Some orgs wanted to see the link in the side menu - check it out in the sites file